### PR TITLE
Reduce reliance on globals

### PR DESCRIPTION
**Pull request by tmc**
Reduce the amount of global shared state (lowering FLAGS use in src/sybil).

### CODE REVIEW

File: src/cmd/cmd_query.go [hunk 1]
```diff
@@ -270,13 +291,13 @@ func RunQueryCmdLine() {
 		sybil.Debug("USING QUERY SPEC", querySpec)
 
 		start := time.Now()
-		// We can load and query at the same time
```
Comment (okayzed) [inline src/cmd/cmd_query.go]: you don't like my comments? :P

File: src/sybil/table_query.go [hunk 2]
```diff
@@ -311,7 +316,7 @@ func (t *Table) LoadAndQueryRecords(loadSpec *LoadSpec, querySpec *QuerySpec) in
 		mu.Unlock()
 		count += rowStoreQuery.count
 
-		if !DELETE_BLOCKS_AFTER_QUERY && t.RowBlock != nil {
+		if loadSpec != nil && loadSpec.SkipDeleteBlocksAfterQuery && t.RowBlock != nil {
```
Comment (okayzed) [inline src/sybil/table_query.go]: how is loadSpec nil again?

Comment (tmc) [reply src/sybil/table_query.go]: in the current codebase it won't be. I generally prefer making called code avoid panics but I don't have a strong preference.

File: src/cmd/cmd_query.go (hunk 1 above)
Comment (tmc) [reply src/cmd/cmd_query.go]: heh, cherry pick conflict resolution artifact

**Pull request merged**